$MirOS: ports/databases/postgresql/patches/patch-src_timezone_pgtz_c,v 1.1.7.1 2005/03/18 15:44:16 tg Exp $
--- src/timezone/pgtz.c.orig	Wed Nov 22 00:11:32 2006
+++ src/timezone/pgtz.c	Thu Feb 22 20:25:47 2007
@@ -310,10 +310,10 @@ score_timezone(const char *tzname, struc
 		{
 			elog(DEBUG4, "TZ \"%s\" scores %d: at %ld %04d-%02d-%02d %02d:%02d:%02d %s versus %04d-%02d-%02d %02d:%02d:%02d %s",
 				 tzname, i, (long) pgtt,
-				 pgtm->tm_year + 1900, pgtm->tm_mon + 1, pgtm->tm_mday,
+				 (int)pgtm->tm_year + 1900, pgtm->tm_mon + 1, pgtm->tm_mday,
 				 pgtm->tm_hour, pgtm->tm_min, pgtm->tm_sec,
 				 pgtm->tm_isdst ? "dst" : "std",
-				 systm->tm_year + 1900, systm->tm_mon + 1, systm->tm_mday,
+				 (int)systm->tm_year + 1900, systm->tm_mon + 1, systm->tm_mday,
 				 systm->tm_hour, systm->tm_min, systm->tm_sec,
 				 systm->tm_isdst ? "dst" : "std");
 			return i;
